src/Data/ConfessionData.re

```diff
@@ -0,0 +1,103 @@
+type confessionComment = {
```

Comment:
personally I often like to actually make these type declarations into separate files in one folder (so a separate file for each type declaration)

The biggest reason is it makes it really easy to find am typedef through the IDE file search, and makes developing a lot more pleasurable.

In this current style I'd have to comb through this file line by like (or use in file search) but it's a bit of a hassle, and will only get worse as the project grows